cli/admin/src/main/java/io/pravega/cli/admin/controller/metadata/ListControllerMetadataEntriesCommand.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.controller.metadata;
+
+import com.google.common.base.Preconditions;
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.utils.AdminSegmentHelper;
+import io.pravega.cli.admin.serializers.controller.ControllerMetadataSerializer;
+import io.pravega.client.tables.impl.HashTableIteratorItem;
+import io.pravega.client.tables.impl.TableSegmentEntry;
+import io.pravega.shared.protocol.netty.PravegaNodeUri;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+import static io.pravega.cli.admin.serializers.controller.ControllerMetadataSerializer.isStreamMetadataTableName;
+
+public class ListControllerMetadataEntriesCommand extends ControllerMetadataCommand {
```

Comment:
Shouldn't this be `ControllerMetadataListEntriesCommand` ?

Reply:
Changed.